packages/compose/package.json

```diff
@@ -21,6 +21,7 @@
 	"main": "build/index.js",
 	"module": "build-module/index.js",
 	"react-native": "src/index",
+	"sideEffects": false,
```

Comment:
I looked for any things in this package that would violate `sideEffects` and didn't find any 👍 

> A "side effect" is defined as code that performs a special behavior when imported, other than exposing one or more exports. An example of this are polyfills, which affect the global scope and usually do not provide an export.

[source](https://webpack.js.org/guides/tree-shaking/#mark-the-file-as-side-effect-free)